conda-recipe/build_cmake.py

```diff
@@ -0,0 +1,92 @@
+import os
+import subprocess
+
+if os.environ["ONEAPI_ROOT"] != None:
+    ONEAPI_ROOT = os.environ["ONEAPI_ROOT"]
+    os.system("source" + " " + ONEAPI_ROOT + "/compiler/latest/env/vars.sh")
+    os.environ["CC"] = "clang"
+    os.environ["CXX"] = "clang++"
+else:
+    print("DPCPP is needed to build DPPL. Abort!")
+
+build_cmake_dir = "build_cmake"
+path = os.path.join(os.getcwd(), build_cmake_dir)
+if os.path.exists(path):
+    os.rmdir(path)
+os.mkdir(path)
+
+INSTALL_PREFIX = path + "/../install"
+if os.path.exists(INSTALL_PREFIX):
+    os.rmdir(INSTALL_PREFIX)
+
+import distutils.sysconfig
+import numpy
+
+PYTHON_INC = distutils.sysconfig.get_python_inc()
+NUMPY_INC = numpy.get_include()
+DPCPP_ROOT = ONEAPI_ROOT + "/compiler/latest/linux/"
+
+# subprocess.check_call(
+#     [
+#         "cmake",
+#         "-DCMAKE_BUILD_TYPE=Release",
+#         "-DCMAKE_INSTALL_PREFIX=" + INSTALL_PREFIX,
+#         "-DCMAKE_PREFIX_PATH=" + INSTALL_PREFIX,
+#         "-DDPCPP_ROOT=" + DPCPP_ROOT,
+#         "-DPYTHON_INCLUDE_DIR=" + PYTHON_INC,
```

Comment:
Please check the latest `build_for_develop.sh` I removed PYTHON_INC and NUMPY__INC as these are not used.